services/js-poc/deno_fleek/src/ops.rs

```diff
@@ -0,0 +1,331 @@
+use serde_json::json;
+
+// Here
+#[op2(fast)]
+pub fn op_set_raw(
+    _state: &mut OpState,
+    _rid: u32,
+    _is_raw: bool,
+    _cbreak: bool,
+) -> anyhow::Result<(), AnyError> {
+    unimplemented!()
+}
+
+#[op2(fast)]
+pub fn op_can_write_vectored(_state: &mut OpState, #[smi] _rid: ResourceId) -> bool {
+    unimplemented!()
+}
+
+#[op2(async)]
+#[number]
+pub async fn op_raw_write_vectored(
+    _state: Rc<RefCell<OpState>>,
+    #[smi] _rid: ResourceId,
+    #[buffer] _buf1: JsBuffer,
+    #[buffer] _buf2: JsBuffer,
+) -> anyhow::Result<usize, AnyError> {
+    unimplemented!()
+}
+
+#[op2]
+#[serde]
+pub fn op_bootstrap_unstable_args(_state: &mut OpState) -> Vec<String> {
+    unimplemented!()
+}
+
+#[op2]
+pub fn op_http_set_response_trailers(
+    _external: *const c_void,
+    #[serde] _trailers: Vec<(ByteString, ByteString)>,
+) {
+    unimplemented!()
+}
+
+#[op2(fast)]
+pub fn op_bootstrap_color_depth(_state: &mut OpState) -> i32 {
+    unimplemented!()
+}
+
+#[derive(Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CreateWorkerArgs {
+    _has_source_code: bool,
+    _name: Option<String>,
+    _permissions: Option<ChildPermissionsArg>,
+    _source_code: String,
+    _specifier: String,
+    _worker_type: WebWorkerType,
+    _close_on_idle: bool,
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum WebWorkerType {
+    Classic,
+    Module,
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub struct WorkerId(u32);
+
+#[op2]
+#[serde]
+pub fn op_create_worker(
+    _state: &mut OpState,
+    #[serde] _args: CreateWorkerArgs,
+    #[serde] _maybe_worker_metadata: Option<JsMessageData>,
+) -> anyhow::Result<WorkerId, AnyError> {
+    unimplemented!()
+}
+
+#[op2]
+pub fn op_host_post_message(
+    _state: &mut OpState,
+    #[serde] _id: WorkerId,
+    #[serde] _data: JsMessageData,
+) -> anyhow::Result<(), AnyError> {
+    unimplemented!()
+}
+
+#[op2(async)]
+#[serde]
+pub async fn op_host_recv_ctrl(
+    _state: Rc<RefCell<OpState>>,
+    #[serde] _id: WorkerId,
+) -> anyhow::Result<WorkerControlEvent, AnyError> {
+    unimplemented!()
+}
+
+#[op2(async)]
+#[serde]
+pub async fn op_host_recv_message(
+    _state: Rc<RefCell<OpState>>,
+    #[serde] _id: WorkerId,
+) -> anyhow::Result<Option<JsMessageData>, AnyError> {
+    unimplemented!()
+}
+
+#[op2]
+pub fn op_host_terminate_worker(_state: &mut OpState, #[serde] _id: WorkerId) {
+    unimplemented!()
+}
+
+#[op2(reentrant)]
+pub fn op_napi_open<NP, 'scope>(
+    _scope: &mut v8::HandleScope<'scope>,
+    _isolate: *mut v8::Isolate,
+    _op_state: Rc<RefCell<OpState>>,
+    #[string] _path: String,
+    _global: v8::Local<'scope, v8::Object>,
+    _buffer_constructor: v8::Local<'scope, v8::Function>,
+    _report_error: v8::Local<'scope, v8::Function>,
+) -> std::result::Result<v8::Local<'scope, v8::Value>, AnyError>
+where
+    NP: NapiPermissions + 'static,
```

Comment:
can remove the generic i think as per above comment on removing deno_napi dependency
```suggestion
#[op2(reentrant)]
pub fn op_napi_open<'scope>(
    _scope: &mut v8::HandleScope<'scope>,
    _isolate: *mut v8::Isolate,
    _op_state: Rc<RefCell<OpState>>,
    #[string] _path: String,
    _global: v8::Local<'scope, v8::Object>,
    _buffer_constructor: v8::Local<'scope, v8::Function>,
    _report_error: v8::Local<'scope, v8::Function>,
) -> std::result::Result<v8::Local<'scope, v8::Value>, AnyError>
```